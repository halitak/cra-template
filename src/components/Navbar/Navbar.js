import React from 'react';

const Navbar = () => {
  return (
    <nav>
      <span>link 1</span>
      <span>link 2</span>
    </nav>
  );
};

export default Navbar;
