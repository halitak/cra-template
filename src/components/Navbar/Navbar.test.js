import React from 'react';
import { shallow } from 'enzyme';

import Navbar from './Navbar';

it('renders correctly', () => {
  const wrapper = shallow(<Navbar />);
  expect(wrapper).toMatchSnapshot();
});
