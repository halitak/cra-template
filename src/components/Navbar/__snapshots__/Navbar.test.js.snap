// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav>
  <span>
    link 1
  </span>
  <span>
    link 2
  </span>
</nav>
`;
